apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/devops.md

```diff
@@ -1,11 +1,129 @@
+
+### Create your deployments using the Sitecore CLI
+
+The Sitecore CLI has a series of Plugins available to enable different functionality. The Cloud Plugin is created to allow you to interact with XM Cloud and perform the provisioning activities normally completed through the Deploy Application. To perform the same process detailed above you would issue the following CLI commands to achieve the same output.
+
+```
+dotnet sitecore cloud login
+```
+
+This command will open a browser window allowing you to authenticate your CLI with the XM Cloud Deploy application.
+
+```
+dotnet sitecore project create -n <<PROJECT_NAME>>
+```
+
+This command will create a new project with the name provided as the value for the -n parameter. This command will return the ID of the created project which we will use in subsequent commands.
+
+```
+dotnet sitecore cloud environment create --project-id <<PROJECT_ID>> -n << ENVIRONMENT_NAME>>
+```
+
+This command will create a new environment with the name passed into the -n parameter. It will return the ID of the created environment which we will use in subsequent commands.
+
+```
+dotnet sitecore cloud deployment create --environment-id << ENVIRONMENT_NAME >> --upload
+```
+
+This command will create a new deployment for the previously created environment. The `--Upload` flag is optional, when not provided you will get a clean XM Cloud installation, none of your code is deployed.
+
+When running the command within a DevOps pipeline, you can use it to upload your custom codebase to the platform.
+
+### Deploying your head
+
+Sitecore XM Cloud is based on a Headless architecture pattern. This means that your site is served by a separate application (or head), to your main XM Cloud Infrastructure. The customer will need to provision this application as well. Currently, XM Cloud has a connector for Vercel allowing you to provision your head directly from within the XM Cloud Deploy Application. It is recommended to use this connector to deploy your site to Vercel where possible, however there are some scenarios where that won’t be possible and you will have to manage the deployment of this separately, for example
+
+- You’re not using Vercel to host your head.
+- You’re not using a Monorepo and your head and XM Cloud application elements are stored in separate repositories.
+- You’re using the CLI to deploy XM Cloud as described above, and want to manage your head deployments using the CI/CD tooling.
+
+### Deploy using the Vercel Connector
+
+If you can use the Vercel Connector then once you have provisioned your XM Cloud Infrastructure and created a Site return to the Sites tab in the Deploy application for your Environment, and click the button on the right and choose “Set up hosting”. Complete the sections and follow the prompts to deploy your site to a new Vercel installation
```

Comment:
The first part seems unnecessary, especially given the section title, and it feels a bit long for a sentence. Recommended change:

Once you have provisioned your XM Cloud Infrastructure and created a Site return to the Sites tab in the Deploy application for your Environment. Then, click the button on the right and choose “Set up hosting”. Complete the sections and follow the prompts to deploy your site to a new Vercel installation